src/pfe/portal/modules/User.js

```diff
@@ -736,8 +737,10 @@ module.exports = class User {
 
     // Only update the Kube secret when running in K8s
     let isServiceAccountPatched = false;
+    let serviceAccountPatchData;
```

Comment:
Does the error on line 735 need a specific message key like the other errors that are thrown in this function?

Reply:
Not necessarily, this is for the overall try catch for updating docker config. If there is an exception on a place where i am not throwing, then this will return the error to the IDE caller.